### PR TITLE
fix: PNRR disclaimer

### PR DESCRIPTION
**Pull request by mfortini**

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Update CatalogueHeader.js
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against de8fa988e9a8f01548a2bf75694595989bb661a8
</p>

File: assets/js/components/Catalogue/CatalogueHeader.js [hunk 1]
```diff
@@ -30,8 +30,13 @@ export const CatalogueHeader = React.memo(() => {
...
           <i>
-            Questo software può concorrere al raggiungimento degli obiettivi delle misure indicate, secondo quanto
-            dichiarato da chi lo mette a disposizione.{' '}
+            Questo software può concorrere al raggiungimento degli obiettivi delle misure indicate,{' '}
+            <b>
+              secondo quanto dichiarato da chi lo mette a disposizione.
+              <br />
+              Developers Italia non manutiene direttamente, né garantisce che il software sia sufficiente a rispondere
+              alle previsioni del PNRR.
+            </b>{' '}
```
Comment (ruphy) [inline assets/js/components/Catalogue/CatalogueHeader.js:39]:
```suggestion
                        Questo software può concorrere al raggiungimento degli obiettivi delle misure indicate, secondo quanto
            dichiarato da chi lo mette a disposizione.{' '}
```

File: _layouts/software-details.html [hunk 2]
```diff
@@ -22,6 +22,11 @@
 {% endcomment %}
 
 <div class="container page-software-detail">
+    <div class="row">
+        <div class="alert alert-info col-md-10 offset-md-1" role="alert">
```
Comment (ruphy) [inline _layouts/software-details.html:26]:
```suggestion
        <div class="col-md-10 offset-md-1">
```

File: _data/l10n.yml [hunk 3]
```diff
@@ -207,6 +207,7 @@ it:
       goToPlatform: "Vai alla piattaforma"
 
     software:
+      disclaimerDevelopers: <i>Garantiamo che i software del catalogo sono open source, ma non forniamo supporto o manutenzione, né assicuriamo che rispondano a specifiche esigenze delle pubbliche amministrazioni.  <br /> Le informazioni presenti nelle schede sono fornite da chi pubblica il software.</i>
```
Comment (ruphy) [inline _data/l10n.yml:210]:
```suggestion
      disclaimerDevelopers: <i>Garantiamo che i software del catalogo siano open source, ma non forniamo supporto o manutenzione, né assicuriamo che rispondano a specifiche esigenze delle pubbliche amministrazioni.  <br /> Le informazioni presenti nelle schede sono fornite da chi pubblica il software.</i>
```

File: assets/js/components/Catalogue/CatalogueHeader.js [hunk 4]
```diff
@@ -43,6 +48,15 @@ export const CatalogueHeader = React.memo(() => {
       <div className="col-10 mx-auto text-center mb-3">
         <SearchBar onChange={handleSearch} defaultValue={searchValue} placeholder={l10NLabels.search_form_label} />
       </div>
+
+      <div className="alert alert-info text-left" role="alert">
+        <i>
+          Garantiamo che i software del catalogo sono open source, ma non forniamo supporto o manutenzione, né
```
Comment (ruphy) [inline assets/js/components/Catalogue/CatalogueHeader.js:54]:
```suggestion
          Garantiamo che i software del catalogo siano open source, ma non forniamo supporto o manutenzione, né
```

File: assets/js/components/Catalogue/CatalogueHeader.js [hunk 5]
```diff
@@ -43,6 +48,15 @@ export const CatalogueHeader = React.memo(() => {
       <div className="col-10 mx-auto text-center mb-3">
         <SearchBar onChange={handleSearch} defaultValue={searchValue} placeholder={l10NLabels.search_form_label} />
       </div>
+
+      <div className="alert alert-info text-left" role="alert">
```
Comment (ruphy) [inline assets/js/components/Catalogue/CatalogueHeader.js:52]:
```suggestion
      <div className="text-left">
```

**Review by ruphy — COMMENTED** [review]
ti lascio un paio di commenti fatti a mano dall'interfaccia web

**Pull request closed without merging**